index.js

```diff
@@ -81,7 +81,13 @@ Rollout.prototype.get = function (key, id, opt_values, multi) {
         if (!modifiers[modName].condition) {
           modifiers[modName].condition = defaultCondition
         }
-        output = modifiers[modName].condition(opt_values[modName])
+        try {
+          output = modifiers[modName].condition(opt_values[modName])
+        }
+        catch (err) {
```

Comment:
style nitpick on newline after `try {}`.
Usually `catch` is put on the same line of the ending `}` for the `try`
```
try {
  // code
} catch (err) {
  // error handling
}
```